docs/shader-specifications/1-shaders.md

```diff
@@ -40,6 +42,9 @@ The result of this shader is this:
 
 ![simple_uv_map](https://raw.githubusercontent.com/wolfenrain/umbra/main/assets/simple_uv_map.png)
 
+> **Note**
+> **Adhere to the rules** mentioned [here](https://github.com/flutter/engine/blob/master/lib/spirv/README.md) while writing your GLSL code. Some of these rules are handled by Umbra while generating the raw GLSL code but for some, you need to be on the lookout.
+
```

Comment:
```suggestion
```

I think we can get this removed here now that we have it a the top ;-)

Reply:
sry about that.😅 I thought I had it removed!
Pushed the changes.